students/jemielity_kamil/lesson_06_dicts_tuples_sets_args_kwargs/args_inspector.py

```diff
@@ -0,0 +1,19 @@
+def inspect_args(*args, **kwargs):
+    print("***** List of arguments: ******")
+    counter = 1
+    for arg in args:
+        print("Argument " + str(counter) + ": " + str(arg))
+        counter += 1
+
+    print()
+    print("***** List of kwargs: ******")
+    counter = 1
+    for kwarg_key, kwarg_value in kwargs.items():
+        print("Kwarg " + str(counter) + ": " + str(kwarg_key) +
+              " value: " + str(kwarg_value))
+        counter += 1
```

Comment:
Above implementation is fine.

You might also use enumerate to skip explicit counter incrementation
```
    print("***** List of kwargs: ******")
    for counter, (kwarg_key, kwarg_value) in enumerate(kwargs.items()):
        print("Kwarg " + str(counter) + ": " + str(kwarg_key) +
              " value: " + str(kwarg_value))
```